usr/lib/common/key.c

```diff
@@ -3185,6 +3185,335 @@ CK_BBOOL kea_priv_check_exportability(CK_ATTRIBUTE_TYPE type)
     return TRUE;
 }
 
+//  dilithium_publ_set_default_attributes()
+//
+CK_RV dilithium_publ_set_default_attributes(TEMPLATE *tmpl, CK_ULONG mode)
```

Comment:
Would it make sense to change the function name of those dilithium functions to include 'ibm' as part of name? When official PKCS11 support for dilithium comes at some point in time, we might want to distinguish between the IBM specific key type and the PKCS11 key type. The PKCS11 key type will most likely have different attributes.

Reply:
ok, I renamed the 6 attribute-related functions, the dilithium_generate_keypair function, and also the version-checking related constants by pre-pending an "ibm_". I re-tested after these changes to make sure I didn't forget a code piece.